test/contentset.test.js

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const { codes } = require('../src/SDKErrors')
+
+beforeEach(() => {
+    fetchMock.reset()
+})
+test('createContentSet - program-not-found', async () => {
+    fetchMock.mock({ 'https://cloudmanager.adobe.io/api/program/123/contentSets' : 'POST', name: 'POST-https://cloudmanager.adobe.io/api/program/123/contentSets' }, 404)
+
+    expect.assertions(2)
+    const sdkClient = await createSdkClient()
+
+    const result = sdkClient.createContentSet('123', '{}')
+
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).rejects.toEqual(
+        new codes.ERROR_CREATE_CONTENTSET({ messageValues: 'https://cloudmanager.adobe.io/api/program/123/contentSets (404 Not Found)'}),
+    )
+})
+
+test('createContentSet - bad-request', async () => {
+    fetchMock.mock({
+                       'https://cloudmanager.adobe.io/api/program/123400/contentSets': 'POST',
```

Comment:
Normally, a linter should have kicked in (not sure if properly configured in this project), but we should fix the indentation to align code.

Example:
```
fetchMock({
    key: value,
    key2: {
       key3: value
    }
});
```

There are other similar issues in the 2 test files. Please adapt those manually, for now.

Reply:
Configured IntelliJ javascript style to 2 spaces, which is what the code looks to be using.